tensorboard/webapp/notification_center/_views/notification_center_component.ng.html

```diff
@@ -0,0 +1,24 @@
+<!--
+@license
+Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
nit: 2021?

Reply:
Thanks for catch this! Done (and for all other files)